src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,75 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        Injector injector = Injector.getInstance("mate.jdbc");
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(1L);
```

Comment:
you shouldn't set id for not created object, fix this, the same for other similar cases in this method